api/test/integration/projects/update-projects.spec.ts

```diff
@@ -38,11 +33,10 @@ describe('Update projects', () => {
     const createProjectDto: CreateProjectDto = {
       countryCode: 'IND',
       projectName: 'Conservation project',
-      ecosystem: ECOSYSTEM.MANGROVE,
+      ecosystem: ECOSYSTEM.SEAGRASS,
```

Comment:
I updated this because we have NO DATA for any national average loss rate for Mangroves, we need to check this with science ASAP